plugins/jobtype/src/azkaban/jobtype/HadoopHiveJob.java

```diff
@@ -22,12 +22,15 @@
 import java.io.IOException;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
```

Comment:
I was think this as a clean up i.e. every jobtype takes care of clean up. As discussed offline killing from command line is a admin specific case moreover its hard to capture hard kill.